repos/base-hw/src/core/include/spec/x86_64/board.h

```diff
@@ -34,4 +34,4 @@ namespace Genode
 	};
 }
 
-#endif /* _CORE__INCLUDE__SPEC__X86__BOARD_H_ */
```

Comment:
As addition: we can of course get rid of the distinction between x86 and x86_64 by moving everything to x86_64. As long as we do not support x86_32 in base-hw (hopefully never ;-)) this is a rational reason from my point of view. But than we should to this consistently and in an extra issue. Do you agree?